controllers/tasks.js

```diff
@@ -19,6 +19,19 @@ const addNewTask = async (req, res) => {
       ...req.body,
       createdBy,
     };
+
+    // Retrieve the issue assignee's RDS user info
+    if (Object.keys(body).includes("github")) {
+      if (Object.keys(body.github.issue).includes("assignee")) {
+        const { user } = await fetchUser({ githubUsername: body.github.issue.assignee });
+        body.github.issue.assigneeRdsInfo = {
+          username: user.username ?? "",
+          firstName: user.first_name ?? "",
+          lastName: user.last_name ?? "",
```

Comment:
Yes, but if n tasks that were converted from issue and all n had an issue assignee, making n separate calls to get all n user's username would have been an overload. Hence, I added the issue assignee information to the task.

Reply:
So tomorrow if a user updates their details then won't this info be out of sync, How should we handle this type of problem, any suggestions?

Reply:
One way could be to fetch user details (if required) when task details are fetched. This would be a better solution than having separate calls for tasks from UI.

Reply:
Yes, we can do this

I think we are already doing that, we can do the same here also

Reply:
Done.